public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -184,10 +199,14 @@ const BannerTestVariantContentEditor: React.FC<BannerTestVariantContentEditorPro
               required: EMPTY_ERROR_HELPER_TEXT,
               validate: templateValidator,
             })}
-            error={errors.messageText !== undefined}
-            helperText={errors.messageText ? errors.messageText.message : BODY_DEFAULT_HELPER_TEXT}
+            error={errors.paragraphs !== undefined}
+            helperText={
+              errors.paragraphs && errors.paragraphs[0]
```

Comment:
Another 'not happy with' comment - this time I'm not sure about how the error checker records error messages for an array of copy. Need to check with how it works with the Epic paragraphs array code in main branch.

Reply:
For the epic it's actually maintained as a string up until `onSubmit` - see [here](https://github.com/guardian/support-admin-console/blob/main/public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx#L119).

We'll change that when the new RichTextEditor component is added, but for now I suggest following what the epic does. This unfortunately involves having a [separate type](https://github.com/guardian/support-admin-console/blob/main/public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx#L58) for the `useForm`.

Reply:
Banner body copy now follows Epic functionality